department-of-veterans-affairs/abd-vro
# add a directory for domain-cc and a couple requirements

## What was the problem?
Contention classification team needs a place to put source code.

Associated tickets or Slack threads:
- [VA.gov Claim Classification set up directory task](https://github.com/orgs/department-of-veterans-affairs/projects/873#)

## How does this fix it?[^1]
Developer can clone and run sample endpoint.

## How to test this PR
- Just getting started on a prototype, this probably doesn't merit testing
- Follow instructions in the [CC readme](https://github.com/department-of-veterans-affairs/abd-vro/blob/feature-add-domain-cc/domain-cc/README.md) if you want to run fastAPI for fun


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
